join.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: page
+title: Join Hackland
+description: 
+---
+
+Members can use the space whenever they want, 24 hours every day of the week.
+
+The hackspace is regularly open to the public whenever a member is in to open up, and weekly on Thursday from 6pm - this is the busiest time and the best for new people visiting.
+
+Membership costs $20 a week but please pay as much as you feel is fair so the space can survive.
+
+Membership includes full access to everything 24/7* including:
```

Comment:
Suggest putting a * where the footnote/conditions to the 24/7 are written. Is it the Other things to note section?

Reply:
Fixed